spek-junit-platform-engine2/src/main/kotlin/org/spekframework/junit/SpekEngineDescriptor.kt

```diff
@@ -0,0 +1,9 @@
+package org.spekframework.junit
```

Comment:
This is a major change, i. e. changing the package name. Shouldn’t it be only in a major version update?